department.rb

```diff
@@ -0,0 +1,37 @@
+class Department
+  attr_reader :dep_name, :dep_employees, :salary, :evaluation
+  def initialize(dep_name)
+    @dep_name = dep_name
+    @dep_employees = []
+    @total_salary = []
+  end
+
+  def add(employee)
+    employee.department = @dep_name
+    @dep_employees << employee
+  end
+
+  def salary
+    @dep_employees.each do |s|
+      @total_salary << s.salary
+    end
+    @total_salary.reduce(:+) #{|sum, salary| sum += salary}
+  end
+
+  def dept_raise(budget)
```

Comment:
Actually, you could just call it "raise", as it's already in the context of a department.